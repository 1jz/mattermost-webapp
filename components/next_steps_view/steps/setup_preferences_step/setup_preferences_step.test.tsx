// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
// See LICENSE.txt for license information.
import React from 'react';

import {shallow} from 'enzyme';

import {TestHelper} from 'utils/test_helper';

import SetupPreferencesStep from './setup_preferences_step';

describe('components/next_steps_view/steps/setup_preferences_step', () => {
    const props = {
        id: 'test',
        currentUser: TestHelper.getUserMock(),
        onSkip: () => { },
        onFinish: () => {}
    };

    test('should match snapshot', () => {
        const wrapper = shallow(
            <SetupPreferencesStep {...props}/>
        );
        expect(wrapper).toMatchSnapshot();
    });
});
