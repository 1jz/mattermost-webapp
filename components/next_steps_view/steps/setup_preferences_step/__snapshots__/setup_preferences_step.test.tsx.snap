// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/next_steps_view/steps/setup_preferences_step should match snapshot 1`] = `
<Fragment>
  <TextCardWithAction
    buttonDefaultMessage="Set Preferences"
    buttonMessageId="next_steps_view.preferenceSetup.setPreferences"
    cardBodyDefaultMessage="You can change how you receive notifications, update your profile, customize display settings and more. Preferences can be accessed through the Main Menu."
    cardBodyMessageId="next_steps_view.preferenceSetup"
    onClick={[Function]}
  />
</Fragment>
`;
